items.py

```diff
@@ -19,6 +17,24 @@ def sort_pubkey(key):
         }
     }
 
+svc_systemd = {
+    'ssh': {
+        'enabled': True,
+        'running': True,
+    }
+}
+
+sign_host_keys = {}
+if node.metadata.get('openssh').get('sign_host_keys').get('enabled'):
+    conf = node.metadata.get('openssh').get('sign_host_keys')
+
+    for host_key in conf.get('keys'):
+        sign_host_keys[f'{node.hostname}_sign_ssh_{host_key}'] = {
```

Comment:
would still go with a key of '/etc/sshd/id_rsa.pub' since this will be unique, and we can signe different keys as well.